ui/app/components/role-edit.js

```diff
@@ -81,7 +81,8 @@ export default Component.extend(FocusOnInsertMixin, {
     createOrUpdate(type, event) {
       event.preventDefault();
 
-      const modelId = this.model.id;
+      // all of the attributes with fieldValue:'id' are called `name`
+      const modelId = this.model.id || this.model.name;
```

Comment:
is there a reason to keep `this.model.id` here? In case somehow the id differs from name on edit?

Reply:
ID does exist if we're editing, so I figured we'd keep it as the authoritative source of truth if it exists